test/examples/Analytical/box_gamma_mixture.jl

```diff
@@ -8,25 +8,26 @@ include("../utils/plotting_helpers.jl")
 FT = Float64
 
 # Initial condition
-moment_init = [100.0, 10.0, 2, 1e-6, 1e-5, 2e-4]
-# 100/cm^3
+moment_init = [1e8, 1e-2, 2e-12, 1, 1e-8, 2e-16]
 dist_init = [
-    GammaPrimitiveParticleDistribution(FT(100), FT(0.1), FT(1)),    # 100/cm^3; 10^5 µm^3; k=1
-    GammaPrimitiveParticleDistribution(FT(1e-6), FT(10), FT(1)),   # 0/cm^3; 10^7 µm^3; k=1
+    GammaPrimitiveParticleDistribution(FT(1e8), FT(1e-10), FT(1)),
+    GammaPrimitiveParticleDistribution(FT(1), FT(1e-8), FT(1)),
 ]
 
 # Solver
-kernel_func = (x, y) -> 5e-3 * (x + y)
-kernel = CoalescenceTensor(kernel_func, 1, FT(500))
+kernel_func = (x, y) -> 5 * (x + y)
```

Comment:
Again, let's document the implied SI units here for clarify. What are the units on (x, y) and the units on 5 ?